### PR TITLE
[Shop] Add template events in shop panel

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets | related #7717 |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
@pamil WDYT?

File: src/Sylius/Bundle/ShopBundle/Resources/views/Account/AddressBook/create.html.twig [hunk 1]
```diff
@@ -9,6 +9,8 @@
             <div class="sub header">{{ 'sylius.ui.add_address'|trans }}</div>
         </h1>
 
+        {{ sonata_block_render_event('sylius.shop.account.address.book.create.after_content_header', {'address': address}) }}
```
Comment (pamil) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Account/AddressBook/create.html.twig]: `address.book` -> `address_book`? (here and everywhere else)

File: src/Sylius/Bundle/ShopBundle/Resources/views/Account/changePassword.html.twig [hunk 2]
```diff
@@ -19,10 +19,15 @@
             {{ 'sylius.ui.change_password'|trans }}
             <div class="sub header">{{ 'sylius.ui.set_a_new_password_for_your_account'|trans }}</div>
         </h1>
+
+        {{ sonata_block_render_event('sylius.shop.account.change.password.after_content_header', {'form': form}) }}
```
Comment (pamil) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Account/changePassword.html.twig]: `change.password` -> `change_password`?

File: src/Sylius/Bundle/ShopBundle/Resources/views/Account/requestPasswordReset.html.twig [hunk 3]
```diff
@@ -12,10 +12,15 @@
                     {{ 'sylius.ui.reset_password'|trans }}
                     <div class="sub header">{{ 'sylius.ui.set_a_new_password_for_your_account'|trans }}</div>
                 </div>
+
+                {{ sonata_block_render_event('sylius.shop.request.password.reset.token.after_content_header') }}
```
Comment (pamil) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Account/requestPasswordReset.html.twig]: `request.password.reset.token` -> `request_password_reset_token`?

File: src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/selectPayment.html.twig [hunk 4]
```diff
@@ -5,23 +5,36 @@
...
             <div class="ui padded segment" id="sylius-payment-methods">
+                {{ sonata_block_render_event('sylius.shop.checkout.select.payment.before_form', {'order': order}) }}
+
                 {{ form_start(form, {'action': path('sylius_shop_checkout_select_payment'), 'attr': {'class': 'ui loadable form', 'novalidate': 'novalidate'}}) }}
                     <input type="hidden" name="_method" value="PUT" />
 
                     {% include '@SyliusShop/Checkout/SelectPayment/_form.html.twig' %}
                     <div class="ui hidden divider"></div>
+
+                    {{ sonata_block_render_event('sylius.shop.checkout.select.payment.before_navigation', {'order': order}) }}
```
Comment (pamil) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/selectPayment.html.twig]: `select.payment` -> `select_payment`?

**Comment by pjedrzejewski** [comment]
Thank you Grzesiu! 👍

**Pull request merged**